spec/support/matchers.rb

```diff
@@ -1,3 +1,6 @@
+require 'rspec/expectations'
```

Comment:
I'm thinking this require should be inside the `rspec/matchers/fail_matchers` file...